### ISSUE TITLE
Examples are too old.

### ISSUE THREAD

**Issue by Misfit-John** [jaredhanson/passport-google-oauth#97]
This is not a really bug or problem.

I just try to use this passport. But I found out that it only works for 3.5.x express(I don't sure, but 3.5.1 works for me) I have also tried another repo [passport-google-oauth2](https://github.com/mstade/passport-google-oauth2) . But this repo fail to get pass the req.isAuthenticated to me... So I just give up.

I think it's time to update this repo to use latest express... Or maybe you can change the package.json file to tell us that this example can only run on 3.5.x express.

P.s. this repo is great.Thank you so much.

**Comment by danrpts** [comment]
Agreed. This project is awesome but I'd like to see some of these open pull requests merged to update the example code. #81 updates example to use with express 4

**Comment by adelphes** [comment]
Agreed. The examples, although well-written, are completely useless from a fresh clone - requiring significant updates to use the latest versions of express.
